src/Client.php

```diff
@@ -135,13 +157,13 @@ public function __construct(Configuration $config, ResolverInterface $resolver =
      * @param string|null $base
      * @param array       $options
      *
-     * @return \GuzzleHttp\ClientInterface
```

Comment:
Using fully qualified names in phpdoc makes it human readable as well as machine readable.

Reply:
I don't really agree; it makes the code harder to read and the overwhelming convention in the community is to use the imported names

Having said that, the `Client` <-> `ClientInterface` could be confusing; solved in 664e9de20ea2142d875427e23e4fa02496e43cf9

Reply:
The community only do this because they have IDEs written by Java programmers. Laravel, for example, uses fully qualified class names, only.

Reply:
Also, importing a class only to use it in the phpdoc actually has a runtime performance hit in PHP, unlike in Java.